packages/api-client/src/main/team/team/NewTeamData.ts

```diff
@@ -20,8 +20,13 @@
 import {MemberData} from '../member/MemberData';
 
 export interface NewTeamData {
+  /** User binding team */
+  binding?: boolean;
+  /** Initial team member IDs (between 1 and 127) */
```

Comment:
Are the IDs between 1-127 or are there 1-127 batched team member data sets?

Reply:
The IDs are between 1 and 127.